pkg/kubelet/metrics/metrics.go

```diff
@@ -462,6 +462,26 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// GracefulShutdownStartTime is a gauge that records the time at which the kubelet is started to graceful shutdown.
+	GracefulShutdownStartTime = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           "graceful_shutdown_start_time",
```

Comment:
As per Prometheus naming conventions, metrics should have a suffix describing their unit: https://prometheus.io/docs/practices/naming/#metric-names
```suggestion
			Name:           "graceful_shutdown_start_time_seconds",
```